### ISSUE TITLE
MapLibre JS section on maplibre.org website does not respect prefers-reduced-motion or WCAG 2.0

### ISSUE THREAD

**Issue by CharlesBelov** [maplibre/maplibre-gl-js#3974]
**maplibre-gl-js version**: not applicable

**browser**: Firefox 124.0.2

### Steps to Trigger Behavior

Case 1: Prefers reduced motion
 1. Set operating system to Reduce Motion (Mac)
 2. Visit https://maplibre.org/
 3. Find the homepage section about MapLibreJS

Case 2: WCAG 2.0 [2.2.2 Pause, Stop, Hide](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#time-limits-pause) 
 1. Set operating system to not Reduce Motion (Mac)
 2. Visit https://maplibre.org/
 3. Find the homepage section about MapLibreJS

### Link to Demonstration
https://maplibre.org/

### Expected Behavior
1. (Case 1) The image does not animate. There ideally is a play button, but failing that, the image does not animate.
1. (Case 2) The image only animates for 5 seconds or has a visible pause button

### Actual Behavior
1. (Case 1) The image animates for more than 5 seconds. There is no visible way to pause it.
2. (Case 2) The image animates for more than 5 seconds. There is no visible way to pause it.

**Comment by HarelM** [comment]
Thanks for taking the time to repost this issue! 
I'll move it to the relevant repo.
Cc: @wipfli
